app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = I18n.t "application.applicationtittle"
```

Comment:
1. Xóa ` = ""` đi hì
2. Phân biệt cho chị `blank?` và `empty?` khác nhau thế nào nhé
ở đây thì dùng blank? đi hì

Reply:
Dạ blank thì có thể có \r \n hay khoảng cách,.... (chỉ xài cho string) 
Còn empty thì dùng đc cho các collection (ko đc dùng cho Enumerable) check xem length của nó có > 0 ko ạ.
.empty? can be used on strings, arrays and hashes and returns true if
.blank? also evaluates true on strings which are non-empty but contain only whitespace
https://stackoverflow.com/questions/885414/a-concise-explanation-of-nil-v-empty-v-blank-in-ruby-on-rails